exercises/concept/phone-number-analysis/.docs/instructions.md

```diff
@@ -0,0 +1,32 @@
+# Instructions
+
+This exercise has you analyze phone numbers.
+
+You are asked to implement 2 features.
+
+Phone numbers passed to the routines are guaranteed to be in the form
+NNN-NNN-NNNN e.g. 212-515-9876.
+
+## 1. Analyze a phone number
+
+Your analysis should return 3 pieces of data
+
+1. An indication of whether the number has a New York dialing code ie. 212 as the first 3 digits
+2. An indication of whether the number is fake having 555 as a prefix code in positions 5 to 7 (numbering from 1)
```

Comment:
I would switch feature ##1 Analyze a phone number and ##2 to Detect fake prefix code.
Simply because the second part of feature 1 relies on feature 2.